skia/renderer/build/premake5.lua

```diff
@@ -2,47 +2,51 @@ workspace "rive"
 configurations {"debug", "release"}
 
 project "rive_skia_renderer"
-kind "StaticLib"
-language "C++"
-cppdialect "C++17"
-toolset "clang"
-targetdir "bin/%{cfg.buildcfg}"
-objdir "obj/%{cfg.buildcfg}"
-includedirs {"../include", "../../../include", "../../dependencies/skia", "../../dependencies/skia/include/core",
+    kind "StaticLib"
+    language "C++"
+    cppdialect "C++17"
+    toolset "clang"
+    targetdir "%{cfg.system}/bin/%{cfg.buildcfg}"
+    objdir "%{cfg.system}/obj/%{cfg.buildcfg}"
```

Comment:
@umberto-sonnino @mjtalbot This now builds the libraries and objects into platform specific folders. So the macosx/bin/rive.a, ios/bin/rive.a, etc.  

Downstream we'll need to update scripts to target the correct directory for .a files. For example, on macosx you'll need to update the libdirs (or any -L cflag that was referencing one of these dirs) to macosx/bin/. 

The rive-wasm and rive-ios branches that point to this branch are already up to date, but be mindful if we merge this that we'll need to either merge those changes too or cherry pick them before updating rive-cpp.